Template/Components/Sermon Row.html

```diff
@@ -1,10 +1,10 @@
 <tr>
-    <td class="fixedsizecell">[*Date*]</td>
-    <td>[*SermonTitle*]</td>
-    <td>[*Passage*]</td>
+    <td class="fixedsizecell"><date></td>
```

Comment:
`<date>`, `<sermonTitle>`, `<passage>` and `sermonSource` are custom pseudo‐nodes defined in your sermon row unfolder.